sys/log/full/src/log_nmgr.c

```diff
@@ -220,18 +247,29 @@ log_nmgr_encode_entry(struct log *log, struct log_offset *log_offset,
      * inside.
      */
     g_err |= cbor_encoder_create_indef_byte_string(&rsp, &str_encoder);
-    for (off = 0; off < len && !g_err; ) {
-        rc = log_read_body(log, dptr, data, off, sizeof(data));
-        if (rc < 0) {
-            g_err |= 1;
-            break;
-        }
+    if (too_long) {
+        data[0] = 0;
```

Comment:
This line (`data[0] = 0;`) is not necessary, is it?  `sprintf()` always null-terminates the destination string, so I think this can be removed.  This same comment applies to the deprecated `LOG_VERSION` case below.

Reply:
Yeah, I do this always with string stuff when I do not need to. Addressed the issues. Thanks for the review!